src/lib/api.js

```diff
@@ -52,6 +49,8 @@ module.exports = function (opts, callback) {
                 opts.uri = parts[0] + '?' + qs;
 
             }
+        } else {
+            opts['X-Requested-With'] = 'XMLHttpRequest';
```

Comment:
1) I think you meant `opts.headers['X-Requested-With']`
2) Might be better to do  

```javascript
if (opts.host === '') {
  defaults(opts.headers, {'X-Requested-With': 'XMLHttpRequest'});
}
```
at the top where we set up defaults. That way we could override this if we wanted to with `{headers: {'X-Requested-With': null}}` if we wanted to.